[export] Add additional tlparse logging

angelayi opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #146939
* #146955
* #146859
* #146858
* __->__ #146534
* #146533
* #146532

Added some additional logging so we can also run tlparse on generic export errors

cc @ezyang @SherlockNoMad @EikanWang @jgong5 @wenzhe-nrv

angelayi committed:
[symbolic shape] Add dtrace for data dependent error

[ghstack-poisoned]

angelayi committed:
Update on "[symbolic shape] Add dtrace for data dependent error"




cc ezyang SherlockNoMad EikanWang jgong5 wenzhe-nrv

[ghstack-poisoned]

angelayi committed:
Update on "[symbolic shape] Add dtrace for data dependent error"




cc ezyang SherlockNoMad EikanWang jgong5 wenzhe-nrv

[ghstack-poisoned]

angelayi committed:
Update on "[export] Add additional tlparse logging"


Added some additional logging so we can also run tlparse on generic export errors

cc ezyang SherlockNoMad EikanWang jgong5 wenzhe-nrv

[ghstack-poisoned]

pianpwk reviewed (approved):
coooooooooooooool (pending tests)

angelayi committed:
Update on "[export] Add additional tlparse logging"


Added some additional logging so we can also run tlparse on generic export errors

cc ezyang SherlockNoMad EikanWang jgong5 wenzhe-nrv

[ghstack-poisoned]

angelayi commented on torch/export/_draft_export.py:
```diff
@@ -214,7 +214,7 @@ def _hash(self, element: tuple[str, dict[str, Any]]) -> int:
             return hash((key, json.dumps(data["stack"])))
         elif key == "create_unbacked_symbol":
-            return hash((key, json.dumps(data["stack"])))
+            return hash((key, json.dumps(data["user_stack"])))
```
oops

angelayi committed:
Update on "[export] Add additional tlparse logging"


Added some additional logging so we can also run tlparse on generic export errors

cc ezyang SherlockNoMad EikanWang jgong5 wenzhe-nrv

[ghstack-poisoned]

angelayi committed:
Update on "[export] Add additional tlparse logging"


Added some additional logging so we can also run tlparse on generic export errors

cc ezyang SherlockNoMad EikanWang jgong5 wenzhe-nrv

[ghstack-poisoned]
